Clock Divider incompatible with latest dev Rack

Hi Iohann ! The latest changes in Rack have left your Clock Divider in a bit of a puzzle. Here's the problem I've hit :

[dlphilp@The6300 VCVRack-Simple]$ make
g++ -I"." -fPIC -I../../include -I../../dep/include -MMD -g -O3 -march=nocona -ffast-math -Wall -Wextra -Wno-unused-parameter -DARCH_LIN -std=c++11 -c -o build/src/ClockDivider.cpp.o src/ClockDivider.cpp
src/ClockDivider.cpp:147:7: error: virtual function ‘virtual void ClockDivider::initialize()’
  void initialize() override
       ^
In file included from ../../include/rack.hpp:6:0,
                 from src/Simple.hpp:3,
                 from src/ClockDivider.cpp:1:
../../include/engine.hpp:62:15: error: overriding final function ‘virtual void rack::Module::initialize()’
  virtual void initialize() final {}
               ^
../../compile.mk:52: recipe for target 'build/src/ClockDivider.cpp.o' failed
make: *** [build/src/ClockDivider.cpp.o] Error 1


The latest Rack code has dropped initialize() and is now using reset(), which causes a problem with the Clock Divider code. I'm not a programmer, and I have almost no experience debugging C++, so I'm open to any suggestions you might have for a fix to the problem. I like your plugins. :)

--- a/src/ClockDivider.cpp
+++ b/src/ClockDivider.cpp
@@ -114,12 +114,6 @@ struct ClockDivider : rack::Module
 		};
 	}
 
-	void reset()
-	{
-		std::for_each(m_clockDividers.begin(), m_clockDividers.end(),
-					  [](ClockDividerImp& imp) { imp.reset(); });
-	}
-
 	void step() override
 	{
 		auto const& inputClock = inputs.at(INPUT_CLOCK);
@@ -144,12 +138,13 @@ struct ClockDivider : rack::Module
 		return m_clockDividers[i].lightState();
 	}
 
-	void initialize() override
+	void reset() override final
 	{
-		reset();
+		std::for_each(m_clockDividers.begin(), m_clockDividers.end(),
+					  [](ClockDividerImp& imp) { imp.reset(); });
 	}
 
-	void randomize() override
+	void randomize() override final
 	{
 		reset();
 	}

